pkg/gen/typegen.go

```diff
@@ -918,3 +923,43 @@ func additionalSecretOutputs(gvk schema.GroupVersionKind) []string {
 		return []string{}
 	}
 }
+
+func aliasesForGVK(gvk schema.GroupVersionKind) []string {
```

Comment:
Is there any reason we can't just alias all versions for a given resource kind?

Reply:
Not necessarily. The initial thought was to limit the scope to things we've checked, but we should be able to make this more general in the future.

Reply:
https://github.com/pulumi/pulumi-kubernetes/issues/828